src/main/java/org/folio/integration/http/VertxOkapiHttpClient.java

```diff
@@ -31,7 +36,8 @@ public CompletableFuture<Response> get(String path,
 
     final var request = client
       .get(url.getPort(), url.getHost(), url.getPath())
-      .putHeaders(buildHeaders(okapiHeaders));
+      .putHeaders(buildHeaders(okapiHeaders))
+      .timeout(5000);
```

Comment:
Can a test be written demonstrating the behaviour of the client when the timeout is exceeded (maybe using WireMock)?

(This might lead to introducing a field for the timeout and an optional constructor parameter, so the tests can specify a smaller timeout).

Reply:
I've no idea, I've never tried to write a test to do so.  I'll do some searching and see if I can figure it out.

Reply:
There is a similar kind of test in mod-circulation - https://github.com/folio-org/mod-circulation/blob/8c6602f37a5b5c64c7f8388f8f5e3aacb7b5496f/src/test/java/org/folio/circulation/support/http/client/VertxWebClientOkapiHttpClientTests.java#L208